DataDefinitions/CommodityDefinition.cs

```diff
@@ -18,6 +18,7 @@ static CommodityDefinition()
             // Check https://eddb.io/archive/v5/commodities.json (for any undefined EDDBID's or undefined FDevID's) and https://github.com/EDCD/FDevIDs (for any undefined FDevID's)
             var _ = new List<CommodityDefinition>
             {
+                new CommodityDefinition(0, 0, "Unknown", Unknown, 0, false),
```

Comment:
Is there a more descriptive name we can give this? Something that indicates that we are waiting to determine what kind of cargo is needed for this mission?

Reply:
IMO, 'Unknown' is plenty descriptive and can be used for multiple contexts.

VB's ResourceBasedLocalizedEDName constructor already returns 'unknown' for null values. I simply 'legitimized' 'Unknown' as an accepted EDName value.